globals.gd

```diff
@@ -770,8 +792,28 @@ class progress:
 		if globals.main && lostRope > 0:
 			globals.main.infotext(str(lostRope) + ' rope%s wore out from use' % ('s' if lostRope > 1 else ''),'red')
 		return numPersons - lostRope
-	
 
+func addrelations(person, person2, value):
```

Comment:
the bugfix changes the start of this function to be
`func addrelations(person, person2, value):
	if person == player || person2 == player || person == person2:
		return
	if person == null || person2 == null:
		return
	if person.relations.has(person2.id) == false:`

Reply:
Fixed